includes/admin/shortcodes/shortcode-give-donor-wall.php

```diff
@@ -46,17 +46,28 @@ public function define_fields() {
 				'tooltip'     => esc_attr__( 'Select a Donation Form', 'give' ),
 				'placeholder' => '- ' . esc_attr__( 'Select a Donation Form', 'give' ) . ' -',
 			),
+			array(
+				'type'        => 'listbox',
+				'name'        => 'order',
+				'label'       => esc_attr__( 'Columns:', 'give' ),
+				'tooltip'     => esc_attr__( 'Sets the number of forms per row.', 'give' ),
+				'options'     => array(
+					'DESC' => esc_html__( 'Newest to Oldest', 'give' ),
+					'ASC'  => esc_html__( 'Oldest to Newest', 'give' ),
+				),
+				'placeholder' => esc_html__( 'Newest to Oldest', 'give' ),
+			),
 			array(
 				'type'        => 'textbox',
-				'name'        => 'donors_per_page',
+				'name'        => 'number',
 				'label'       => esc_attr__( 'Donors Per Page', 'give' ),
 				'placeholder' => '20',
```

Comment:
Let's make the default donors per page `12` since it is divisible by 2, 3, and 4 which means it is most likely to create a complete final row on each page. I also think this creates the best balance of quantity and aesthetics. Showing more than `12` by default would take up a lot of real estate.